d8/d8batch/config/install/node.type.feed_source.yml

```diff
@@ -0,0 +1,17 @@
+uuid: 84b0fdd2-d490-4076-a612-1804aa7aae46
+langcode: en
+status: true
+dependencies:
+  module:
+    - menu_ui
```

Comment:
Probably, because the entity (content type) contains menu_ui settings. See below. I'll check what will happen if I remove these from the automatically generated file.